src/Loaders/LoadingSpinner.tsx

```diff
@@ -25,18 +25,9 @@ interface Props {
   size?: Size;
 }
 
-// The whole animation is exactly 5 seconds long.
-// Each rotation is 450 degrees, with a deceleration at
-// 420 degrees, and a reduced speed between 420-450.
-// 1st rotation: 60 deg and 90 deg
-// 2nd rotation: 150 deg and 180 deg
-// 3rd rotation: 240 deg and 270 deg
-// 4th rotation: 330 deg and 360 deg (restart loop)
 const SPIN = keyframes`
```

Comment:
nbd: I didn't know we wanted to get rid of the fancy animation for all loaders, I thought it was only for loader icons. 👍assuming that switch did happen but wanted to confirm

Reply:
Good question @jgzuke ; see this https://meteor.slack.com/archives/CFSCN3G7L/p1567213272006100